sidebars.js

```diff
@@ -131,6 +131,14 @@ module.exports = {
           type: "link",
           href: "/blog/integrate-apache-spark-questdb-time-series-analytics",
         },
+        {
+          label: "SQLAlchemy",
+          customProps: {
+            tag: "New",
+          },
+          type: "link",
+          href: "/docs/third-party-tools/sqlalchemy",
```

Comment:
this code should work but the intended way to link between docs pages is this:

```suggestion
          customProps: {
            tag: "New",
          },
          type: "doc",
          id: "third-party-tools/sqlalchemy",
```

Reply:
Question - should I add a trailing slash to these, similar to all the hyperlinks we have been amending?

```suggestion
          label: "SQLAlchemy",
          customProps: {
            tag: "New",
          },
          type: "link",
          href: "/third-party-tools/sqlalchemy/",
```

Reply:
That's optional, but for consistency it would be nice